keras_hub/src/models/retinanet/prediction_head.py

```diff
@@ -0,0 +1,148 @@
+import keras
+
+
+class PredictionHead(keras.layers.Layer):
+    """The classification/box predictions head.
+
+    Arguments:
+        output_filters: int. Number of convolution filters in the final layer.
+        num_filters: int. Number of convolution filters used in base layers.
+            Defaults to `256`.
+        num_conv_layers: int. Number of convolution layers before final layer.
+            Defaults to `4`.
+        kernel_initializer: `str` or `keras.initializers`.
+            The kernel initializer for the convolution layers.
```

Comment:
go to next line after line limit.  This line should continue on the above line - `The kernel initializer...`

Reply:
Done